corehq/apps/email/models.py

```diff
@@ -0,0 +1,150 @@
+import jsonfield
+from django.db import models
+from django.db import transaction
+
+from corehq.apps.email.util import BadEmailConfigException, get_email_backend_classes
+from corehq.util.quickcache import quickcache
+
+
+class EmailSMTPBackend(models.Model):
```

Comment:
I believe we also need a configurable `from_email`

Reply:
+1 to needing a `from_email` as a user input and in the model.

Reply:
I agree that one sounds reasonable (we pretty much only use `settings.DEFAULT_FROM_EMAIL` for stuff coming from our gateway).  To use more we'd need some configuration system to decide which one to use when.